demos/README.md

```diff
@@ -1,75 +1,219 @@
 # Demos integration
 
-Technical demos folder to test wiris plugins with different types of editors.
+Technical demos is an internal project developed to test, finding bugs and improvements, and validate the wiris plugins with different types of editors.
+
+## Table of contents
+
+- [Requirements](#requirements)
+- [Instructions](#instructions)
+  - [Cloning](#cloning)
+  - [Quick start](#quick-start)
+    - [Bootstraping](#bootstraping)
+    - [Installation](#installation)
+    - [Run a demo with the public npm package](#run-a-demo-with-the-public-npm-package)
+    - [Run a demo with the local package](#run-a-demo-with-the-local-package)
+    - [Use an extension to check that the files follow the standards](#use-an-extension-to-check-that-the-files-follow-the-standards)
+- [File manifest](#file-manifest)
+  - [Important files and folder structure](#important-files-and-folder-structure)
+  - [Supported editors](#supported-editors)
+  - [Technologies with a demo](#technologies-with-a-demo)
+- [Known bugs](#known-bugs)
+- [Change log](#change-log)
+- [News](#news)
+- [Documentation](#documentation)
+- [License](#license)
+- [Contact information](#contact-information)
+
+## Requirements
+
+This project requires basic knowledge of the `git` and `npm` commands.
+It also uses the `npx` tool which comes bundled with `npm`.
+
+To install node and npm follow this [guide-npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm), and use this other one [guide-git](https://www.atlassian.com/git/tutorials/install-git) to install git.
+
+
+[Lerna](https://lerna.js.org/) is used as a cornerstone of the project structure. Its main features, and how to install it, are explained in this guide.
 
 ## Instructions
 
-First, follow the instructions of the README file you can find in the root directory of the project.
+Please, follow the instructions to have the best experience when working with this project.
```

Comment:
Because the only documentation that remains the same as in the README of the root is the cloning, the other instructions contain specific variants of the demos or are completely new.